compiler/modules/front/expr.bal

```diff
@@ -106,9 +106,12 @@ class ExprContext {
         return resolveSubsetTypeDesc(self.mod, self.defn, td);
     }
 
-    function createTmpRegister(bir:SemType t, Position? pos = ()) returns bir:Register {
-        bir:Register reg = bir:createRegister(self.code, t, (), pos);
-        return reg;
+    function createTmpRegister(bir:SemType t, Position? pos = ()) returns bir:TempRegister {
```

Comment:
We need to be consistent on Tmp vs Temp. I suggest we stick with Tmp (i.e. don't use Temp)

Reply:
fixed